react/react1/week3/my-todo-list/src/components/TodoList.js

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect } from 'react';
+import Button from './Button';
+import TodoRow from './TodoRow';
+import AddTodo from './AddTodo';
+import Calendar from './Calendar';
+import BlackBorder from './BorderComponent';
+
+const TodoList = () => {
+  const [date, setDate] = useState('');
+  const [newTodo, setNewTodo] = useState('');
+  const [todosList, setTodosList] = useState([]);
+
+  const API_URL =
+    'https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw';
+
+  const FetchApi = () => {
+    fetch(API_URL)
+      .then((response) => response.json())
```

Comment:
Add some error handling if the fetch is slow or does not fullfill and show some error to user